test/tests/amqp/test_amqp_racknode.py

```diff
@@ -0,0 +1,211 @@
+'''
+Copyright 2017 Dell Inc. or its subsidiaries.  All Rights Reserved.

```

Comment:
Per Jeanne/Tim, this is what legal finally recommended.  Use this on new collateral going forward. :-)
```Copyright (c) 2017 Dell Inc. or its subsidiaries.  All Rights Reserved. ```